tutorials/getting_started/configuration.md

```diff
@@ -0,0 +1,344 @@
+that embeds the output of our text fields, a
+[`Seq2SeqEncoder`](http://docs.allennlp.org/en/latest/api/allennlp.modules.seq2seq_encoders.html#allennlp.modules.seq2seq_encoders.seq2seq_encoder.Seq2SeqEncoder)
+that transforms that sequence into an output sequence,
+and a linear layer to convert the output sequences
+into logits representing the probabilities of predicted tags.
+(The last layer is not configurable and so won't appear in our configuration file.)
+
+### The Text Field Embedder
+
+Let's first look at the text field embedder configuration:
+
+```js
+    "text_field_embedder": {
+            "tokens": {
+                    "type": "embedding",
+                    "embedding_dim": 50
+            },
+            "token_characters": {
+              "type": "character_encoding",
+              "embedding": {
+                "embedding_dim": 8
+              },
+              "encoder": {
+                "type": "cnn",
+                "embedding_dim": 8,
+                "num_filters": 50,
+                "ngram_filter_sizes": [5]
+              },
+              "dropout": 0.2
+            }
+    },
+```
+
+You can see that it has an entry for each of the namespaces in our `TextField`.
+Each entry specifies a
+[`TokenEmbedder`](http://docs.allennlp.org/en/latest/api/allennlp.modules.token_embedders.html?highlight=embedding#allennlp.modules.token_embedders.token_embedder.TokenEmbedder)
+that indicates how to embed
+the tokens in that particular namespace. The `TextFieldEmbedder`'s output is the concatenation
+of these embeddings.
+
+The `"tokens"` namespace (which consists of integer encodings of the lowercased words in the input)
+gets fed into an
+[`Embedding`](http://docs.allennlp.org/en/latest/api/allennlp.modules.token_embedders.html?highlight=embedding#allennlp.modules.token_embedders.embedding.Embedding)
+module that embeds the vocabulary words in a 50-dimensional space.
+
+The `"token_characters"` namespace (which consists of integer-sequence encodings of the characters in each word)
+gets fed into a
+[`TokenCharactersEncoder`](http://docs.allennlp.org/en/latest/api/allennlp.modules.token_embedders.html?highlight=embedding#allennlp.modules.token_embedders.token_characters_encoder.TokenCharactersEncoder),
+which embeds the characters in an 8-dimensional space
+and then applies a
+[`CnnEncoder`](http://docs.allennlp.org/en/latest/api/allennlp.modules.seq2vec_encoders.html#allennlp.modules.seq2vec_encoders.cnn_encoder.CnnEncoder)
+that uses 50 filters and so also produces a 50-dimensional output. You can see that this encoder also uses a 20% dropout during training.
+
+The output of this `TextFieldEmbedder` is a 50-dimensional vector for `"tokens"`
+concatenated with a 50-dimensional vector for `"token_characters`";
+that is, a 100-dimensional vector.
+
+### The Seq2SeqEncoder
```

Comment:
Why is this a `Seq2SeqEncoder`?  The JSON below has nothing about `Seq2Seq` but rather specifies `stacked_encoder`.  I'm guessing that `stacked_encoder = Seq2SeqEncoder` but it's an implciit connection.